model.py

```diff
@@ -0,0 +1,137 @@
+"""Models for Markov chains Tweet generator app"""
+
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import os
+
+db = SQLAlchemy()
+
+
+class User(db.Model):
```

Comment:
Since you have multiple tables, you can have a parent "model" folder and split these classes into their own files. This will help with readability and if you wanted to add more to your app

Reply:
Awesome! Thanks for the tip. I just separated out the tables into separate files with [this commit](https://github.com/missy-davies/chatterbot/commit/7008d75f6bc7a2fb18dd8d902dd7ed0df99c653c)